js/net/src/group.test.ts

```
@@ -0,0 +1,77 @@
+	expect(group.tryReadFrame()).toBeUndefined();
+	expect(group.done).toBe(true);
+});
+
+test("readable resolves once a frame is buffered", async () => {
+	const group = new Group(0);
+	// No frame yet: readable() must stay pending for an empty, open group.
+	const readable = group.readable();
+	let settled = false;
+	readable.then(() => {
+		settled = true;
+	});
```

Biome (2.5.0)

[error] A "floating" Promise was found, meaning it is not properly handled and could lead to ignored errors or unexpected behavior.

(lint/nursery/noFloatingPromises)